maintainers/maintainer-list.nix

```diff
@@ -4735,6 +4735,12 @@
     githubId = 587870;
     name = "Jonathan Mettes";
   };
+  jo1gi = {
```

Comment:
Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add jo1gi``.